ts/features/it-wallet/navigation/ItwRoutes.ts

```diff
@@ -40,7 +40,8 @@ export const ITW_ROUTES = {
         DATA: "ITW_PRESENTATION_CREDENTIAL_DATA",
         RESULT: "ITW_PRESETATION_CREDENTIAL_RESULT"
       } as const
-    } as const
+    } as const,
+    QRCODE: "ITW_PRESETATION_CROSS_DEVICE_QRCODE"
```

Comment:
Should we move this under PRESENTATION -> GENERIC -> QRCODE and call the screen something like `ItwPrProximityQrCodeScreen.tsx` where `Pr` stands for "Presentation"? It might be verbose but it helps quickly identify a screen. What do you think?